Frontify/guideline-blocks
# fix: Improve editing experience when preview is hidden [CU-8693hvguz]

This PR improves the editing layout when the preview is hidden, according to the design: https://www.figma.com/file/L1O18PAMq8x1Kza5U5a4Hb/Single-Template-Block?type=design&node-id=2545-40497&mode=design&t=INL4ZlBBs0ms0M7v-4

What to test:

- Create template block
- Set the "Preview" setting to "None"
- Make sure the layout looks like the screenshots (or the Figma design). The wording and button is slightly different if there is no template chosen vs. a template chosen.

In the screenshot below:

1. Layout w/o selected template
2. Layout with selected template
3. Old layout

![image](https://github.com/Frontify/guideline-blocks/assets/9266535/1e89c042-2794-4f4b-a13c-78c4a2692f0f)

Mobile view:

![image](https://github.com/Frontify/guideline-blocks/assets/9266535/83896150-bd07-465e-bbc9-02a103f520c7)
